### PR TITLE
Create Expandable Table Row - #3j8nf1w

### PR DESCRIPTION
**Pull request by gitChimp88**
add mockdata and create expandable row from that data, stopping point

### CODE REVIEW

**Review by nunocgoncalves — CHANGES_REQUESTED** [review]
@gitChimp88, This is some really good progress.
That's exactly the logic we want to follow for the table formatting and filtering.

There is only a small bug I came across and I raised a task on ClickUp. Check it out [here](https://app.clickup.com/t/3gupuvt) for more details.

**Comment by gitChimp88** [comment]
@nunocgoncalves Okay I changed the way I was positioning the first column of the dropdown which helps the movement caused by that, but the columns still resize to fit long text (so it doesn't overlap with other columns) we can use a fixed width but then we would need to limit the number of digits or amount of text allowed in each column to avoid overlap.

**Comment by nunocgoncalves** [comment]
@gitChimp88, Let's use 7 digits for every integer and resize the columns accordingly. Example: 6 -> 6.000000; 60,000 -> 60,000.00

File: src/components/ERC20Balance.jsx [hunk 1]
```diff
@@ -1,94 +1,130 @@
...
+
+  const sumBalanceAndValuesForChains = (data) => {
+    return data.map((chain) => {
+      return {
+        ...chain,
+        balance: chain.tokens.reduce((acc, obj) => (acc += obj.balance), 0),
+        value: chain.tokens.reduce((acc, obj) => (acc += obj.value), 0),
+      };
+    });
+  };
```
Comment (nunocgoncalves) [inline src/components/ERC20Balance.jsx:126]: Good work, nice and concise.

File: src/index.css [hunk 2]
```diff
@@ -124,7 +110,7 @@ tr {
 
 th,
 td {
-  padding: 5px 4px 6px 4px;
+  padding: 5px 4px 16px 4px;
```
Comment (nunocgoncalves) [inline src/index.css:113]: Let's covert this values to "em"

File: src/components/reusable/Table.jsx [hunk 3]
```diff
@@ -36,7 +36,24 @@ const ButtonContainer = styled("div")`
...
+  border-collapse: separate;
+  border-spacing: 0;
+  background-color: var(--layout-white);
+  width: 100%;
+  height: 80%;
+  padding: 20px 20px 10px 40px;
+  border: 2px solid var(--brand-blue);
+  border-collapse: separate;
+  border-radius: 41px;
+  border-spacing: 0px;
```
Comment (nunocgoncalves) [inline src/components/reusable/Table.jsx:49]: Let's convert this to "em"

**Comment by nunocgoncalves** [comment]
Task linked: [CU-3j8nf1w Toggleable Dropdown for Table Row](https://app.clickup.com/t/3j8nf1w)

**Comment by nunocgoncalves** [comment]
@gitChimp88, we can close this, right?

**Comment by nunocgoncalves** [comment]
Used in #79

**Pull request closed without merging**